src/locales/ui/ui-fr.yaml

```diff
@@ -56,10 +56,6 @@ entries:
   Une technologie moderne: Une technologie moderne
   Vous n'avez pas encore fait le test.: Vous n'avez pas encore fait le test.
   actions choisies: actions choisies
-  components.ScoreExplanation.text.p1: Voici vos scores de départ, calculés à partir de réponses attribuées à l'avance à chaque question ! Ils évolueront à chaque nouvelle réponse.
-  components.ScoreExplanation.text.p2: Voici vos scores provisoires, ils évoluent à chaque nouvelle réponse !
-  components.ScoreExplanation.text.p3: 'Si vous répondez "je ne sais pas" à une question, le score ne changera pas : une valeur par défaut vous est attribuée.'
-  components.ScoreExplanation.text.p4: Nous améliorons le calcul et ses valeurs par défaut <2>tous les mois</2>!
```

Comment:
Changes not in en and es

Reply:
Le script de nettoyage des clés inutilisées est mal foutu ; je crée une carte